lib/lib.gradle.kts

```
@@ -89,9 +92,9 @@ moshi {
 }
 
 mavenPublishing {
-    publishToMavenCentral(automaticRelease = true)
+    publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL, automaticRelease = true)
     signAllPublications()
-    coordinates(groupId, artifactId, project.version.toString())
+    coordinates(groupId, artifactId)
```

Do you need to set this to 
```suggestion
    coordinates(groupId, artifactId, project.version)
```

---

Not needed anymore